qutebrowser/browser/downloads.py

```diff
@@ -373,7 +392,6 @@ def delete(self):
         except OSError:
             log.downloads.exception("Failed to remove partial file")
 
-    @pyqtSlot()
```

Comment:
This is called via a signal when invoked via the context menu (in `downloadview.py`), so the decorator should stay.